jwst/refpix/tests/test_refpix.py

```diff
@@ -351,8 +355,8 @@ def test_do_corrections_subarray_no_oddEven(setup_subarray_cube):
     input_model.data[0, 0, :, :] = dataval
     input_model.data[0, 0, :4, :] = bottom_rpix
     input_model.data[0, 0, :, :4] = left_rpix
-    input_model.pixeldq[:4, :] = 2147483648
-    input_model.pixeldq[:, :4] = 2147483648
+    input_model.pixeldq[:4, :] = dqflags.pixel['REFERENCE_PIXEL'])
```

Comment:
There's an extra `)` at the end of lines 358, 359, 401 and 402.  This is why the tests are failing.

Reply:
Thanks, I went ahead and removed all of the extra `)`'s.